docs/testing-guide.md

```diff
@@ -130,6 +132,92 @@ export const BUTTON_COMPONENT = '[data-component="button"]';
 export const BUTTON_SUBTEXT = '[data-element="subtext"]';
 ```
 
+
+## Functional Browser Testing (Playwright)
+
+We are planning to use [Playwright](https://playwright.dev) framework to test component behaviour that requires a browser environment. Functionality which has already been tested via Jest tests does not need to be tested again using Playwright, unless it would be beneficial to test the behaviour in a manner similar to how a user would in a browser.
+
+Further details on installing Playwright and our configuration for it can be found in our [Getting started with Playwright](../playwright/README.md) guide.
+
+### Playwright File Structure
+
+All Playwright tests must go within `*.pw.tsx` for the relevant component.
+
+```
+.
+├── src/
+│   ├── components/
+│   │   └── [component-name]/
+│   │       └── [component-name].pw.tsx
+│   │       └──  components.test-pw.tsx
+│
+├── playwright/
+│   ├── components/
+│   │   └── [component-name]/
+│   │       └── locators.ts
+│   │       └── index.ts
+│   │
+│   ├── support/
+│   │   └── helper.ts
+│   │
+│   ├── index.html
+│   ├── index.tsx
+│   └── README.md
+│
+└── playwright-ct.config.ts
+```
+
+A typical `*.pw.tsx` file may look like the following:
+
+```tsx
+// inside src/components/button/button.pw.tsx
+import { test, expect } from "@playwright/experimental-ct-react17";
+import Button from "./button.component";
+import { buttonComponent } from "../../../playwright/component/button/index";
+
+  test.describe("Check props for Button component", async () => {
+    test("should render Button label when passed to the component", async ({ mount, page }) => {
+
+      const label = "foobar";
+      
+      await mount(<Button>{label}</Button>);
+
+      await expect(await buttonComponent(page)).toHaveText(label);
+    });
+  });
+```
+
+Where `mount` renders the component in the real browser (`chromium`/`webkit`/`firefox`/`opera`) and `buttonComponent` is a _locator_ that returns the DOM element we want to test.
+
+### Locators
+
+We write dedicated functions to access rendered DOM elements in order to make our tests easier to read. Locators for a component typically follow this structure:
+
+`index.ts`
+```ts
+/* in index.ts */
+import { type Page } from "@playwright/test";
```

Comment:
I think this location is incorrect. There's an error in the `index.ts` file stating "Cannot find module '@playwright/test' or its corresponding type declarations."